options.go

```diff
@@ -69,6 +69,170 @@ const (
 	SkipAnyCorruptedRecordsRecovery      = WALRecoveryMode(3)
 )
 
+type Ticker uint32
+
+// Options for use in GetTickerCount and GetAndResetTickerCount
+// For definitions of individual flags, see <rocksdb/statistics.h>
+const (
```

Comment:
These consts will have zero value. Take a look at [iota](https://github.com/golang/go/wiki/Iota) if you need every subsequent constant incremented.

Reply:
Changed to use iota.